### PR TITLE
Add documentation for TiDB specific SQL functions

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### What is changed, added or deleted? (Required)

Add documentation for SQL functions `TIDB_PARSE_TSO` and `TIDB_BOUNDED_STALENESS`.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5699
- Other reference link(s): https://github.com/pingcap/tidb/pull/8385

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/tidb-functions.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+1 row in set (0.00 sec)
+```
+
+### MySQL 兼容性
+
+`TIDB_DECODE_KEY` 是 TiDB 的特有函数，和 MySQL 不兼容。
+
+## TIDB_DECODE_PLAN
+
+`TIDB_DECODE_PLAN` 函数可以用来解码 TiDB 执行计划。你可以在慢查询日志中找到这些计划。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:46]:
```suggestion
`TIDB_DECODE_PLAN` 函数可用来解码 TiDB 执行计划。你可以在慢查询日志中找到 TiDB 执行计划。
```

File: functions-and-operators/tidb-functions.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    TableReader_7          root         319.04     data:Selection_6                           960        time:713.1µs, loops:2, cop_task: {num: 1, max: 568.5µs, proc_keys: 0, rpc_num: 1, rpc_time: 549.1µs, copr_cache_hit_ratio: 0.00}    3.99 KB    N/A
+    └─Selection_6          cop[tikv]    319.04     lt(test.t.a, 10000)                        960        tikv_task:{time:313.8µs, loops:960}                                                                                                   N/A        N/A
+      └─TableFullScan_5    cop[tikv]    960        table:t, keep order:false, stats:pseudo    960        tikv_task:{time:153µs, loops:960}                                                                                                     N/A        N/A
+```
+
+### MySQL 兼容性
+
+`TIDB_DECODE_PLAN` 是 TiDB 的特有函数，和 MySQL 不兼容。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:73]:
```suggestion
`TIDB_DECODE_PLAN` 是 TiDB 特有的函数，和 MySQL 不兼容。
```

File: functions-and-operators/tidb-functions.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+## TIDB_IS_DDL_OWNER
+
+`TIDB_IS_DDL_OWNER` 函数用来检查你连接的 TiDB 实例是否是 DDL Owner。DDL Owner 是代表集群中的所有其他节点执行 DDL 语句的 TiDB 实例。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:77]:
```suggestion
`TIDB_IS_DDL_OWNER` 函数用来检查你连接的 TiDB 实例是否是 DDL Owner。DDL Owner 是代表集群中所有其他节点执行 DDL 语句的 TiDB 实例。
```

File: functions-and-operators/tidb-functions.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+### 另请参阅
+
+- [ADMIN SHOW DDL](/sql-statements/sql-statement-admin-show-ddl.md)
+- [ADMIN CANCEL DDL](/sql-statements/sql-statement-admin-cancel-ddl.md)
+
+## TIDB_PARSE_TSO
+
+`TIDB_PARSE_TSO` 函数可以用来从 TiDB TSO 时间戳中提取物理时间戳。
+
+TSO 指 Time Stamp Oracle，是 PD (place Driver) 为每个事务提供的单调递增的时间戳。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:116]:
```suggestion
TSO 指 Time Stamp Oracle，是 PD (Placement Driver) 为每个事务提供的单调递增的时间戳。
```

File: functions-and-operators/tidb-functions.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+TSO 是一串包括以下两部分的数字：
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:118]:
```suggestion
TSO 是一串数字，包含以下两部分：
```

File: functions-and-operators/tidb-functions.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+```sql
++-----------------------------------+
+| TIDB_PARSE_TSO(@@tidb_current_ts) |
++-----------------------------------+
+| 2021-05-26 11:33:37.776000        |
++-----------------------------------+
+1 row in set (0.0012 sec)
+```
+
+这里使用 `TIDB_PARSE_TSO` 从 `tidb_current_ts` 会话变量可用的时间戳编号中提取物理时间戳。因为每个事务都会给出时间戳，所以此函数在事务中运行。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:149]:
```suggestion
以上示例使用 `TIDB_PARSE_TSO` 函数从 `tidb_current_ts` 会话变量提供的可用时间戳编号中提取物理时间戳。因为每个事务都会分配到时间戳，所以此函数在事务中运行。
```

File: functions-and-operators/tidb-functions.md [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+
+`TIDB_PARSE_TSO` 是 TiDB 的特有函数，和 MySQL 不兼容。
+
+### 另请参阅
+
+- [`tidb_current_ts`](/system-variables.md#tidb_current_ts)
+
+## TIDB_VERSION
+
+`TIDB_VERSION` 函数可以用来获取你连接的 TiDB 服务器的版本和内部版本详细信息。你可以在 GitHub 上提交 issue 时使用此函数。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:161]:
```suggestion
`TIDB_VERSION` 函数用于获取当前连接的 TiDB 服务器版本和构建详细信息。你向 GitHub 上提交 issue 时，你可使用此函数获取相关信息。
```

File: functions-and-operators/tidb-functions.md [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+GoVersion: go1.13
+Race Enabled: false
+TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
+Check Table Before Drop: false
+1 row in set (0.00 sec)
+```
+
+### MySQL 兼容性
+
+`TIDB_VERSION` 是 TiDB 的特有函数，和 MySQL 不兼容。如果你需要兼容 MySQL，可以使用 `VERSION` 获取版本信息，但结果不包含详细的构建信息。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:194]:
```suggestion
`TIDB_VERSION` 是 TiDB 的特有函数，和 MySQL 不兼容。如果要求兼容 MySQL，可以使用 `VERSION` 获取版本信息，但结果不包含详细的构建信息。
```

**Review by TomShawn — COMMENTED** [review]
Global comments:

+ "TiDB 的特有函数" --> "TiDB 特有的函数"
+ ".. 函数可以用来.." --> ".. 函数用于.."

File: functions-and-operators/tidb-functions.md (hunk 4 above)
Comment (JmPotato) [reply functions-and-operators/tidb-functions.md:116]: 应该是 Timestamp Oracle。

File: functions-and-operators/tidb-functions.md (hunk 6 above)
Comment (JmPotato) [reply functions-and-operators/tidb-functions.md:149]: 因为每个事务都会*被*分配到时间戳。

File: functions-and-operators/tidb-functions.md [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+summary: 学习使用 TiDB 特有的函数。
+---
+
+# TiDB 特有的函数
+
+本文档介绍 TiDB 特有的函数。
+
+## TIDB_BOUNDED_STALENESS
+
+`TIDB_BOUNDED_STALENESS` 是 TiDB 内部的函数。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:12]:
```suggestion
`TIDB_BOUNDED_STALENESS` 是 TiDB 的内部函数。
```

File: functions-and-operators/tidb-functions.md [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+
+## TIDB_DECODE_KEY
+
+`TIDB_DECODE_KEY` 函数可用于将 TiDB 编码的键输入解码为包含 `_tidb_rowid` 和 `table_id` 的 JSON 结构。你可以在一些系统表和日志输出中找到 TiDB 的编码键。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:16]: Global comment: "可用于" --> "用于"

File: functions-and-operators/tidb-functions.md [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+      └─TableFullScan_5    cop[tikv]    960        table:t, keep order:false, stats:pseudo    960        tikv_task:{time:153µs, loops:960}                                                                                                     N/A        N/A
+```
+
+### MySQL 兼容性
+
+`TIDB_DECODE_PLAN` 是 TiDB 特有的函数，和 MySQL 不兼容。
+
+## TIDB_IS_DDL_OWNER
+
+`TIDB_IS_DDL_OWNER` 函数可用于检查你连接的 TiDB 实例是否是 DDL Owner。DDL Owner 是代表集群中所有其他节点执行 DDL 语句的 TiDB 实例。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:77]:
```suggestion
`TIDB_IS_DDL_OWNER` 函数可用于检查你连接的 TiDB 实例是否是 DDL Owner。DDL Owner 代表集群中所有其他节点执行 DDL 语句的 TiDB 实例。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@JmPotato PTAL, thanks!

File: functions-and-operators/tidb-functions.md [hunk 12]
```diff
@@ -0,0 +1,194 @@
...
+summary: 学习使用 TiDB 特有的函数。
+---
+
+# TiDB 特有的函数
+
+本文档介绍 TiDB 特有的函数。
+
+## TIDB_BOUNDED_STALENESS
+
+`TIDB_BOUNDED_STALENESS` 是 TiDB 的内部函数。
```
Comment (JmPotato) [inline functions-and-operators/tidb-functions.md:12]: 具体用法可以链接到 Stale Read 的部分：https://github.com/pingcap/docs-cn/blob/master/as-of-timestamp.md#%E8%AF%AD%E6%B3%95%E6%96%B9%E5%BC%8F

Comment (TomShawn) [reply functions-and-operators/tidb-functions.md:12]:
```suggestion
`TIDB_BOUNDED_STALENESS` 是 TiDB 的内部函数，用于指定一个时间范围。用法为 `TIDB_BOUNDED_STALENESS(t1, t2)`，其中 t1 和 t2 为时间范围的两端，支持使用日期时间和时间函数。使用该函数，

TiDB 会在指定的时间范围内选择一个合适的时间戳，该时间戳能保证所访问的副本上不存在开始于这个时间戳之前且还没有提交的相关事务，即能保证所访问的可用副本上执行读取操作而且不会被阻塞。
```

File: functions-and-operators/tidb-functions.md [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+
+# TiDB 特有的函数
+
+本文档介绍 TiDB 特有的函数。
+
+## TIDB_BOUNDED_STALENESS
+
+`TIDB_BOUNDED_STALENESS` 是 TiDB 的内部函数，用于指定一个时间范围。用法为 `TIDB_BOUNDED_STALENESS(t1, t2)`，其中 t1 和 t2 为时间范围的两端，支持使用日期时间和时间函数。使用该函数。
+
+TiDB 会在指定的时间范围内选择一个合适的时间戳，该时间戳能保证所访问的副本上不存在开始于这个时间戳之前且还没有提交的相关事务，即能保证所访问的可用副本上执行读取操作而且不会被阻塞。
```
Comment (TomShawn) [inline functions-and-operators/tidb-functions.md:14]:
```suggestion
`TIDB_BOUNDED_STALENESS` 是 TiDB 的内部函数，用于指定一个时间范围。用法为 `TIDB_BOUNDED_STALENESS(t1, t2)`，其中 t1 和 t2 为时间范围的两端，支持使用日期时间和时间函数。

使用该函数，TiDB 会在指定的时间范围内选择一个合适的时间戳，该时间戳能保证所访问的副本上不存在开始于这个时间戳之前且还没有提交的相关事务，即能保证所访问的可用副本上执行读取操作而且不会被阻塞。
```

**Review by JmPotato — APPROVED** [review]

**Pull request merged**